eip712-adapter-moshi/src/main/java/pm/gnosis/eip712/adapters/moshi/TypeAdapters.kt

```diff
@@ -0,0 +1,75 @@
+package pm.gnosis.eip712.adapters.moshi
```

Comment:
Should we really put them in the eip712 module? They are not really related, right? (EIP-712 is about signing typed data and these type adapter are more general)

Reply:
I agree. Is there any other module that's close enough? Or should this be in a separate module?

Reply:
Good question ... I thought about the utils module but that would mean we add moshi as a dependency there ... Maybe a separate one is cleaner .... Not sure to be honest

Reply:
Android-common also feels wrong as moshi is not only for android

Reply:
My suggestion would be to maybe add an additional library that follows the nameing of the `:models` module. Something like `:models-moshi`

Reply:
I called it common-adapter-moshi in 7eae9130b0e4cf87825450a5013e8370d8c0c160